### PR TITLE
Update container list

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
feat: adjust tekton container links

This change replaces all tekton tasks containers with 2 new.
CSV generator was updated to accept only two links for tekton containers

Has to be merged after https://github.com/kubevirt/ssp-operator/pull/532
**Release note**:
```
Update list of tekton containers
```

### CODE REVIEW

**Comment by codingben** [comment]
Can you please:

1. Update PR title and remove `[WIP]` (Instead, please convert that PR to draft, it would also disable CI)
2. Update PR description and commit title from `feat: adjust container links` to something that will also mention tekton? (e.g. `feat: adjust tekton container links`)

File: internal/common/environment.go [hunk 1]
```diff
@@ -16,76 +16,27 @@ import (
...
-	DeafultDiskVirtCustomizeIMG = "quay.io/kubevirt/tekton-task-disk-virt-customize:" + TektonTasksVersion
-	DeafultCreateVMIMG          = "quay.io/kubevirt/tekton-task-create-vm:" + TektonTasksVersion
-	DeafultModifyDataObjectIMG  = "quay.io/kubevirt/tekton-task-modify-data-object:" + TektonTasksVersion
-	DeafultCopyTemplateIMG      = "quay.io/kubevirt/tekton-task-copy-template:" + TektonTasksVersion
-	DeafultCleanupVMIMG         = "quay.io/kubevirt/tekton-task-execute-in-vm:" + TektonTasksVersion
-	GenerateSSHKeysIMG          = "quay.io/kubevirt/tekton-task-generate-ssh-keys:" + TektonTasksVersion
-	DefaultVirtioIMG            = "quay.io/kubevirt/virtio-container-disk:v0.59.0"
+	DefaultTektonTasksIMG         = "quay.io/kubevirt/tekton-tasks:" + TektonTasksVersion
+	DeafultTektonTasksDiskVirtIMG = "quay.io/kubevirt/tekton-tasks-disk-virt:" + TektonTasksVersion
+	DefaultVirtioIMG              = "quay.io/kubevirt/virtio-container-disk:v0.59.0"
```
Comment (0xFelix) [inline internal/common/environment.go:27]: Btw. is the fixed `v0.59.0` appropriate here?

**Review by 0xFelix — COMMENTED** [review]
/approve

File: internal/common/environment.go (hunk 1 above)
Comment (ksimon1) [reply internal/common/environment.go:27]: in TTO we have full automation for bumping version of this variable: https://github.com/kubevirt/tekton-tasks-operator/blob/main/.github/workflows/update-virtio-version.yaml

**Review by codingben — APPROVED** [review]
/lgtm

**Pull request merged**